src/hotspot/share/runtime/vmStructs.cpp

```diff
@@ -1022,7 +1022,11 @@
   nonstatic_field(elapsedTimer,                _active,                                       bool)                                  \
   nonstatic_field(InvocationCounter,           _counter,                                      unsigned int)                          \
                                                                                                                                      \
-  nonstatic_field(UpcallStub::FrameData,       jfa,                                           JavaFrameAnchor)
+  nonstatic_field(UpcallStub::FrameData,       jfa,                                           JavaFrameAnchor)                       \
+  nonstatic_field(Mutex,                       _name,                                         const char*)                           \
```

Comment:
Other classes have their own section. Mutex should also have its own section.
```suggestion

  nonstatic_field(Mutex,                       _name,                                         const char*)                           \
```